Makefile

```
@@ -0,0 +1,82 @@
+dev: ## Start in development mode (default)
+	docker compose up -d bot
+
+prod: ## Start in production mode
+	docker compose --profile prod up -d bot-prod
+
+rebuild: ## Rebuild and restart services
+	docker compose down
+	docker compose build --no-cache
+	docker compose up -d
+
+backup: ## Backup MongoDB database
```

checkmake (0.2.2)

[warning] Target body for "backup" exceeds allowed length of 5 (6).

(maxbodylength)